contract/r/gnoswap/gov/doc.gno

```diff
@@ -0,0 +1,2 @@
+// Package gov manages the governance of the gnoswap protocol, including proposal creation, voting, and execution.
+package gov
```

Comment:
We should have detailed docs here

Reply:
[wrote](https://github.com/gnoswap-labs/gnoswap/pull/488/files#diff-d259e6e9f7cbce47cd7770c88dc5b8cebd335a9bc4dfb5098c54a452de4efa8c)